docs/osx/02-how-to-guides/02-plugin-development/07-publication/index.md

```diff
@@ -116,9 +112,6 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     PluginRepoRegistry__factory.createInterface(),
     'PluginRepoRegistered'
   );
-  if (!eventLog) {
```

Comment:
why did we remove this from the example?

Reply:
This happend on the OSX repo and now translated into the developer portal.
The `findEventLog` function now does the `undefined` check internally.